plugins/catalog-backend/src/ingestion/processors/ldap/config.ts

```diff
@@ -141,6 +150,7 @@ const defaultConfig = {
       rdn: 'cn',
       name: 'cn',
```

Comment:
As a side note (not for this PR), I'm wondering name should be normalized, considering `cn` can often have spaces and be a displayName to being with (which fails validation).

Reply:
Some will be able to choose a different source for the name than `cn` then. But yeah I heard that it happens that sometimes there is no great choice for metadata.name at all in the source data.

A normalizer makes sense to add, but I've also heard some (was it you?!) that inserted an extra pre-processor after instead, that did some form of custom normalization.

Reply:
Yep that was me. However, we completed abandoned trying to use the LDAP and are using the GitHub-org instead. Beyond the metadata.name issue there is no group and also no way nice way to filter out other groups. Basically groups in our LDAP are currently purely for permissions.